tests/dialects/test_postgres.py

```diff
@@ -370,6 +370,18 @@ def test_postgres(self):
                 "spark": "TRIM(BOTH 'as' FROM 'as string as')",
             },
         )
+        self.validate_all(
+            "CAST('127.0.0.1/32' AS INET)",
+            read={
+                "postgres": "CAST('127.0.0.1/32' AS INET)",
+            },
+        )
+        self.validate_all(
+            "INET '127.0.0.1/32'",
+            read={
+                "postgres": "CAST('127.0.0.1/32' AS INET)",
```

Comment:
you have it inverted, this should be wrote

Reply:
you can combine these two tests into one

Reply:
I see, the "read" is the input and we're testing the inputs to match the expected output, but we've declared the output first. It makes sense when you know that's what it's doing, but I can also see why I got it backwards. Thanks for the hint.